### ISSUE TITLE
Add label extensions

### ISSUE THREAD

**Issue by codemunkie15** [Nerwyn/ha-nunjucks#3]
**Are you requesting that a [Home Assistant template extension](https://www.home-assistant.io/docs/configuration/templating/) be added? Which one?**
Extensions for labels

**Is your feature request related to a problem? Please describe.**
I'm trying to define CSS styles based on a label being attached to an entity.

**Describe the solution you'd like**
`'Presence activated light' in label_entities(config.entity)`

**Describe alternatives you've considered**
I can do this (requires label_id):
`'presence_activated' in hass.entities[config.entity].labels`

But would prefer to do this (uses label name instead):
`'Presence activated light' in label_entities(config.entity)`

Many thanks

**Comment by Nerwyn** [comment]
For my reference: https://www.home-assistant.io/docs/configuration/templating/#labels

I should work on the next minor version of this package once I'm done with v4 of my remote card.

**Comment by Nerwyn** [comment]
@codemunkie15 which frontend project do you want to test the new functions on? I've added a bunch including 1.3.0 and should be able to release a beta build of ha-nunjucks and a matching beta build on universal remote card or custom card features in a day or two.

**Comment by Nerwyn** [comment]
Bad news - it turns out the hass object field I was using for a lot of labels logic `_labelRegistry` doesn't appear consistently and can't be used. I'm going to have to remove the `label_id` and `label_name` functions and limit these functions to just using the label ID, as the label name doesn't seem to be available otherwise.

**Comment by Nerwyn** [comment]
@codemunkie15 have you gotten around to trying this in the custom features project alpha?

**Comment by Nerwyn** [comment]
Good news @codemunkie15 ! I fell into a rabbit hole and ended up implementing almost all of the Home Assistant extensions (except for some harder to implement obscure ones). After that I came back and took another crack at retrieving the label registry and figured it out (I found and adapted the code that does it in the Home Assistant frontend repo)! Try the [custom features 4.1.0 beta](https://github.com/Nerwyn/service-call-tile-feature/releases/tag/4.1.0-beta.003). All label functions are now working like they do in Home Assistant's backend templating system.
